docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
FWIW, here's an example of what it looks like to embed JSON on the command line: http://docs.aws.amazon.com/cli/latest/userguide/cli-using-param.html